packages/binding/src/definition/definition.ts

```diff
@@ -1,251 +1,167 @@
-import type { PropertyBindingInfoElement } from "../types";
-export const propertyBindingInfoElements: PropertyBindingInfoElement[] = [
-  {
-    name: "path",
-    type: [
-      {
-        kind: "string",
-        dependents: [],
-        notAllowedElements: ["value", "parts"],
-      },
-    ],
-    description: {
-      text: `Path in the model to bind to, either an absolute path or relative to the binding context for the corresponding model; when the path contains a '>' sign, the string preceding it will override the model property and the remainder after the '>' will be used as binding path`,
-      visibility: "Public",
-    },
-  },
-  {
-    name: "value",
-    type: [
-      {
-        kind: "string",
-        dependents: [],
-        notAllowedElements: ["path", "parts"],
-      },
-    ],
-    description: {
-      text: "Since 1.61, defines a static binding with the given value",
-      visibility: "Public",
-    },
-  },
-  {
-    name: "model",
-    type: [
-      {
-        kind: "string",
+import { UI5TypedefProp } from "@ui5-language-assistant/semantic-model-types";
+import { PROPERTY_BINDING_INFO } from "../constant";
+import {
+  BindContext,
+  PropertyBindingInfoElement,
+  BindingInfoName,
+  PropertyType,
+  TypeKind,
+} from "../types";
+const buildType = (property: UI5TypedefProp): PropertyType[] => {
```

Comment:
This part seems to be essentially the same - It's still is hardcoded against a specific list of properties.